sdk/src/main/java/me/digi/sdk/interapp/managers/DMENativeConsentManager.kt

```diff
@@ -118,11 +125,19 @@ class DMENativeConsentManager(val sessionManager: DMESessionManager, val appId:
 
         val error = if (!sessionManager.isSessionValid()) {
             DMEAuthError.InvalidSession()
-        }
-        else when (result) {
+        } else when (result) {
             ctx.getString(R.string.const_result_error) -> {
                 DMELog.e("There was a problem requesting consent.")
-                DMEAuthError.General()
+
+                errorMessage?.let {
```

Comment:
In this instance (because of the nesting), it's probably cleaner to have:
```
if (errorMessage != null && errorCode != null) {
    ....
}
```
Kotlin's smart cast will take care of the nulls in that block for you.

Reply:
Strangely enough I just kept the method and those parameters that were complaining before are no longer complaining, so I assume we're good to go here